_scss/main.scss

```diff
@@ -5,19 +5,19 @@
     opacity: 1;
 }
 
-::-webkit-input-placeholder{ 
+::-webkit-input-placeholder{
     @include placeholder-style;
 }
 
-:-moz-placeholder{ 
+:-moz-placeholder{
     @include placeholder-style;
 }
 
-::-moz-placeholder{ 
+::-moz-placeholder{
     @include placeholder-style;
 }
 
-:-ms-input-placeholder { 
+:-ms-input-placeholder {
```

Comment:
These changes don't relate with the PR, so please undo them. You could make another PR for cleaning up whitespaces.

Reply:
I am sorry but I didn't quite understand what changes are required here??Do you want me to add the whitespaces again??

Reply:
just undo these whitespace changes using `git revert`, they shouldn't show up in the diff.

Reply:
From what I have understood git revert would undo all the changes I have made to other files as well as will it go back to previous state before commit. I think the commit df74b51 has just a one whitespace problem and it would not make much difference. I would like to know whether the scroll works well now after using nth-child or do I need to make changes??

Reply:
Yes, the scroll works fine now. There is a way to interactively revert separate hunks, but don't worry I will do it for you. 

I know whitespaces don't matter much to the code, but any unnecessary change screws up `git blame`, which is why you should tend to avoid it. They seem insignificant here, but in larger orgs and  codebases, they lead to unnecessary headaches.

Reply:
Thank you for solving the whitespace problem.Will you merge the PR after reverting???